SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -206,7 +206,7 @@
     </PropertyGroup>
   </Target>
 
-  <Target Name="CleanProjectFolders" AfterTargets="Clean" BeforeTargets="CreateProjectSpecificDirs" Condition=" $(SonarQubeTempPath) != '' ">
+  <Target Name="CleanProjectFolders" AfterTargets="Clean" Condition=" $(SonarQubeTempPath) != '' ">
```

Comment:
I'll look at adding tests (there aren't any currently)

Reply:
The fix works correctly with BizTalk projects in VS2015 (can't open BizTalk projects in VS2017).